udata/tests/forms/test_image_field.py

```diff
@@ -77,7 +77,7 @@ def test_with_image(self):
             doc.image.save(img, 'image.jpg')
         doc.save()
         form = self.F(None, obj=doc)
-        self.assertEqual(form.image.filename.data, 'image.jpg')
+        self.assertEqual(form.image.filename.data, 'image.png')
```

Comment:
No, this is because the image are now optimized on `save`, and so converted as png as part of the process. This test the initial form data which is populated from the model field which is now the optimized one.